projects/angular/src/forms/STYLES.MD

```diff
@@ -0,0 +1,58 @@
+# FORMS - Style API
+
+## CSS Variables
+
+| CSS Variable Name                               | Description                                                        |
+| ----------------------------------------------- | ------------------------------------------------------------------ |
+| --clr-form-disabled-background-color            | Changes the default background color for disabled forms.           |
```

Comment:
As we agreed, let's remove the "Changes the..." boilerplate prefix.
Please, apply everywhere.